Source/KSCrash/Recording/KSCrash.m

```diff
@@ -141,6 +144,14 @@ + (instancetype) sharedInstance
     return sharedInstance;
 }
 
++ (instancetype) sharedInstanceWithBasePath:(NSString *)basePath
+{
+    dispatch_once(&onceToken, ^{
+        sharedInstance = [[KSCrash alloc] initWithBasePath:basePath];
```

Comment:
It wont let me mark there with a review but static dispatch_once_t onceToken; needs to be deleted from line 139

Reply:
Oh, you're totally right. Probably forgot about that when migrating code 😂

Reply:
Done!